src/dso_api/dynamic_api/serializers.py

```diff
@@ -105,24 +105,19 @@ def to_representation(self, value: DynamicModel):
 
 
 class _RelatedSummaryField(Field):
-    def to_representation(self, value: DynamicModel):
-        count = value.count()
+    def to_representation(self, value: models.Manager):
         request = self.context["request"]
         url = reverse(get_view_name(value.model, "list"), request=request)
-        url_parts = parse.urlparse(url)
-        parent_pk = value.instance.pk
-        filter_name = list(value.core_filters.keys())[0] + "Id"
-        q_params = {parent_pk: filter_name}
+        filter_field = next(iter(value.core_filters.keys()))
+        q_params = {toCamelCase(filter_field + "_id"): value.instance.pk}
+
         # If this is a temporary slice, add the extra parameter to the qs.
         if request.dataset_temporal_slice is not None:
             key = request.dataset_temporal_slice["key"]
-            value = request.dataset_temporal_slice["value"]
-            q_params[key] = value
-        url_parts = url_parts._replace(query=parse.urlencode(q_params))
-        return {
-            "count": count,
-            "href": parse.urlunparse(url_parts),
-        }
+            q_params[key] = request.dataset_temporal_slice["value"]
+
+        query_string = ("&" if "?" in url else "?") + urlencode(q_params)
```

Comment:
Maybe use the shiny new package by @gkoller : https://pypi.org/project/more-ds/

Reply:
Dan moet er een `urlparse` en `urlunparse` gedaan worden. Deze string interpolatie is wmt voldoende en snel in zo'n geval.